src/[sandbox]/examples/timeline.ts

```diff
@@ -1,5 +1,5 @@
-import "@/components/timeline/Timeline";
-import "@/components/timeline/TimelineItem";
+import "../../components/timeline/Timeline";
```

Comment:
again you can continue importing components through absolute paths. just not the types